src/Cli/Microsoft.TemplateEngine.Cli/PostActionProcessors/RemoveJsonPropertyPostActionProcessor.cs

```diff
@@ -0,0 +1,193 @@
+        };
+
+        private static readonly JsonDocumentOptions DeserializerOptions = new()
+        {
+            AllowTrailingCommas = true,
+            CommentHandling = JsonCommentHandling.Skip
+        };
+
+        public override Guid Id => ActionProcessorId;
+
+        internal static Guid ActionProcessorId { get; } = new Guid("73406DFC-1255-4798-86E8-DF66AB9F7A18");
+
+        protected override bool ProcessInternal(
+            IEngineEnvironmentSettings environment,
+            IPostAction action,
+            ICreationEffects creationEffects,
+            ICreationResult templateCreationResult,
+            string outputBasePath)
+        {
+            if (!action.Args.TryGetValue(JsonFileNameArgument, out string? jsonFileName))
+            {
+                Reporter.Error.WriteLine(string.Format(LocalizableStrings.PostAction_ModifyJson_Error_ArgumentNotConfigured, JsonFileNameArgument));
+                return false;
+            }
+
+            IReadOnlyList<string> jsonFiles = FindFilesInCurrentFolderOrParentFolder(environment.Host.FileSystem, outputBasePath, jsonFileName);
+
+            if (jsonFiles.Count == 0)
+            {
+                return true;
+            }
+
+            if (jsonFiles.Count > 1)
+            {
+                Reporter.Error.WriteLine(string.Format(LocalizableStrings.PostAction_ModifyJson_Error_MultipleJsonFiles, jsonFileName));
+                return false;
+            }
+
+            var jsonElementProperties = JsonContentParameters.CreateFromPostAction(action);
+
+            if (jsonElementProperties is null)
+            {
+                return false;
+            }
+
+            JsonNode? newJsonContent = RemoveElementFromJson(
+                environment.Host.FileSystem,
+                jsonFiles[0],
+                jsonElementProperties!.ParentProperty,
+                ":",
+                jsonElementProperties.JsonPropertyName,
+                action);
+
+            if (newJsonContent is not null)
+            {
+                environment.Host.FileSystem.WriteAllText(jsonFiles[0], newJsonContent.ToJsonString(SerializerOptions));
+                Reporter.Output.WriteLine(string.Format(LocalizableStrings.PostAction_ModifyJson_Succeeded, jsonFileName));
+            }
+
+            return true;
+        }
+
+        private static JsonNode? RemoveElementFromJson(IPhysicalFileSystem fileSystem, string targetJsonFile, string? propertyPath, string propertyPathSeparator, string propertyName, IPostAction action)
+        {
+            JsonNode? jsonContent = JsonNode.Parse(fileSystem.ReadAllText(targetJsonFile), nodeOptions: null, documentOptions: DeserializerOptions);
+
+            if (jsonContent is null)
+            {
+                return null;
+            }
+
+            JsonNode? parentProperty = FindJsonNode(jsonContent, propertyPath, propertyPathSeparator);
+
+            if (parentProperty is not null && parentProperty[propertyName] is not null)
```

Comment:
nit: Could simplify as:

```suggestion
            if (parentProperty?[propertyName] is not null)
```